src/App.js

```diff
@@ -0,0 +1,72 @@
+import {
+	Routes,
+	Route,
+} from "react-router-dom";
+import "./App.css";
+import {
+	Header,
+	Footer,
+	Cart,
+	HomePage,
+	ProductListing,
+	Login,
+	Signup,
+	ForgotPassword,
+	UserPage,
+	WishList,
+} from "./components/index";
+import Mockman from "mockman-js";
+import { FilterProvider } from "./context/FilterProvider";
+
+function App() {
+	return (
+		<div className="App">
+			<Header />
+			<Routes>
+				<Route
+					path="/"
+					element={<HomePage />}
+				/>
+				<Route
```

Comment:
I think you should write path like this:            path="/login"       .
Why it is in this way?
is it working?